specification/07-theory_operations.adoc

```diff
@@ -81,10 +81,12 @@ I/O rules apply:
   used to protect the PCIe link between the TVM and the TDI.
 * A single selective IDE stream is established between the physical device and
   its Root Port. All TDIs from the physical device share this single IDE stream.
-* For each physical device from which a TDI is attached to a TVM, the TSM
-  generates, owns and distributes the IDE stream keys to both the physical
-  device and the RP.
-* The TSM configures the RP PCIe IDE extended capability.
+* For each physical device from which a TDI is bound to a TVM, the TSM that
+  manages this TVM also generates, owns and distributes the IDE stream keys to
```

Comment:
Do we support one TSM manages the TVM, and another TSM managers the device SPDM?

Or do we mandate only one TSM must manage both TVM and the corresponding device SPDM?

Reply:
@rsahita @jyao1 I tried to make that clearer with the latest version of this PR. PTAL.